docs/howto_guides/api_gui.md

```diff
@@ -0,0 +1,165 @@
+# Using the NMDC API Graphical User Interface (GUI)
+
+## Retrieving and Submitting Metadata using the ___Find___ and ___Metadata___ API Endpoints
+
+Metadata describing NMDC data (e.g. studies, biosamples, data objects, etc.) may be retrieved or submittied with GET and POST requests, respectively, using the **[NMDC API Graphical User Interface (GUI)](https://api.microbiomedata.org/docs#/)** The API GUI provides an user interface for programmatic access to the NMDC data portal without needing to use the Command Line.
```

Comment:
needs a basic editing pass, e.g. `submittied` -> `submitted`, but I won't point out any more such trivial changes in this review.

Reply:
Fixed in a05a20ed8ee3a92fe14affe9998f6d5839528933

Reply:
Thanks for fixing these @eecavanna!